### PR TITLE
Added in gitignore

### PR DESCRIPTION
**Pull request by briangonzalez**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_store
```
Comment (bennycwong) [inline .gitignore]: !fix sup

Comment (briangonzalez) [inline .gitignore]: !resolved

Comment (bennycwong) [inline .gitignore]: !fix please ignore more stuff

Comment (briangonzalez) [inline .gitignore]: !resolved i did it

Comment (arjansingh) [inline .gitignore]: !fix too much ignored

Comment (briangonzalez) [inline .gitignore]: !resolved @arjansingh This has been resolved....not.

Comment (briangonzalez) [inline .gitignore]: !fix you need to fix this

Comment (briangonzalez) [inline .gitignore]: !resolved

Comment (briangonzalez) [inline .gitignore]: !fix

Comment (briangonzalez) [inline .gitignore]: !resolved

**Comment by briangonzalez** [comment]
!star This works

**Comment by briangonzalez** [comment]
:+1:

**Comment by briangonzalez** [comment]
:-1:

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+tmp/
```
Comment (jasonbosco) [inline .gitignore]: !fix Fix this

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+node_modules/
```
Comment (jasonbosco) [inline .gitignore]: And this !resolve

**Pull request merged**